excercises/panic_R.py

```diff
@@ -0,0 +1,16 @@
+phrase = "Don't panic!"
+plist = list(phrase)
+print(phrase)
+print(plist)
+
+telist = ["D", "'", "n", "i", "c", "!"]
+for letter in telist:
+    plist.remove(letter)
+
+plist.insert(1, plist.pop())
+plist.insert(2, plist.pop(3))
+plist.insert(4, plist.pop())
```

Comment:
Kod tych trzech linijek 10-12 się powtarza. Ofc, nie ma takiego wymogu, ale dla ćwiczenia może mógłbyś spróbować zrobić sobie funkcję move, która brałaby jako argumenty listę oraz indeks początkowy elementu i indeks końcowy i przesuwała element na pożądane miejsce?

Reply:
Odpisuję tu, bo już to wszystko pozmienialiśmy.